frontend/src/components/Pages/MissionPage/MapPosition/MissionMapView.tsx

```diff
@@ -78,6 +79,9 @@ export const MissionMapView = ({ mission }: MissionProps) => {
         [mission.tasks]
     )
 
+    let displayedMapName = mission.map?.mapName.split('.')[0].replace(/[^0-9a-z-A-Z ]/g, ' ')
```

Comment:
What does this Regex do?

Reply:
Likewise for the line below. If the issue is in the map naming, might it be better to rename it at the source?

Reply:
First line slpits the sentence after a "." and removes everything after and only keeps characters from a to z and 0 to 9. Second one capitalizes the first letter of the map name. So for example a map with name ac_002.png would be renamed to Ac 002.